packages/cdktf/test/helper/data-source.ts

```diff
@@ -8,17 +8,20 @@ import {
   AnyMap,
   ComplexObject,
   ComplexList,
+  IResolvable,
 } from "../../lib";
 import { TestProviderMetadata } from "./provider";
-import { stringToTerraform } from "../../lib/runtime";
+import { listMapper, stringToTerraform } from "../../lib/runtime";
 
 export interface TestDataSourceConfig extends TerraformMetaArguments {
   name: string;
+  listBlock?: IResolvable;
 }
 
 export class TestDataSource extends TerraformDataSource {
   public static readonly tfResourceType: string = "test_data_source";
   public name: string;
+  public listBlock?: IResolvable; // TODO: add special type as well?
```

Comment:
Todo left, What special type?

Reply:
Just for testing. Real life bindings don't have an IResolvable here but a type with an interface as well. I think keeping it simple for now is okay as we currently only use the IResolvable case in tests that use this TestDataSource (for the interfaces we use the edge provider tests).

Reply:
reworded the comment to reflect what I just said